_source/_posts/2020-09-15-reintroducing-joel-franusic.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: blog_post
+title: "Reintroducing Joël Franusic"
+author: joel-franusic
+by: advocate
+communities: [python]
+description: "Joël Franusic shares his personal mission statement, with an live example"
+tags: [python, brython]
+tweets:
+- ""
+- ""
+- ""
+image:
```

Comment:
Add tweets if you want this to show up in our "from the archives" tweets. Also, add a social image please. 🙂

Reply:
This should be addressed now. Do we require three tweet suggestions, or are two okay?